### PR TITLE
Feature/495 timer Added Absolute time to finish

### PR DESCRIPTION
**Pull request by lloback**
The absolute time is the END of the test assigned at a QT level. The users can't finish the test after the set time.

### CODE REVIEW

File: src/components/QualifyingTest/Countdown2.vue [hunk 1]
```diff
@@ -102,9 +106,18 @@ export default {
   },
   mounted() {
     const start = new Date(this.startTime);
-    const end = new Date(this.startTime);
+    let end = new Date(this.startTime);
     end.setMinutes(end.getMinutes() + this.duration);
 
+    // #495 Absolute End
+    const absoluteEnd = new Date(this.endTime);
+    const isAbsluteEndBeforeTheEnd = absoluteEnd < end;
```
Comment (warrensearle) [inline src/components/QualifyingTest/Countdown2.vue:114]: Check spelling `isAbsluteEndBeforeTheEnd` --> `isAbsoluteEndBeforeTheEnd`

File: src/components/QualifyingTest/Countdown2.vue [hunk 2]
```diff
@@ -69,6 +69,10 @@ export default {
       type: Date,
       required: true,
     },
+    endTime: {
+      type: Date,
+      required: true,
```
Comment (warrensearle) [inline src/components/QualifyingTest/Countdown2.vue:74]: Could this be an optional prop?

**Review by warrensearle — CHANGES_REQUESTED** [review]
Couple of tweaks

File: src/components/QualifyingTest/Countdown2.vue (hunk 1 above)
Comment (lloback) [reply src/components/QualifyingTest/Countdown2.vue]: done

File: src/components/QualifyingTest/Countdown2.vue (hunk 2 above)
Comment (lloback) [reply src/components/QualifyingTest/Countdown2.vue]: optional with a default value of 
``` default: 'Fri Dec 31 2049 23:59:59 GMT+0000 (Greenwich Mean Time)', ```

**Review by warrensearle — APPROVED** [review]
Looks good 👍 
We need to get this tested both manually and via unit tests

**Pull request merged**